helm-charts/submarine/values.yaml

```diff
@@ -45,3 +45,11 @@ podSecurityPolicy:
 istio:
   enabled: true
   gatewaySelector: ingressgateway
+
```

Comment:
Could you verify that this configuration works? I remember that the configuration of the dependency chart should have name as the root node, e.g:
```yaml
seldon-core-operator:
  executor:
    defaultEnvSecretRefName: submarine-serve-secret 2> /dev/null
  istio:
    enabled: true
``